src/e2e/helpers/constants.ts

```diff
@@ -1,3 +1,5 @@
 import { Direction } from '../../types'
 
 export const NEW_THOUGHT_GESTURE: Direction[] = ['r', 'd']
+
+export const NEW_SUB_THOUGHT_GESTURE: Direction[] = ['r', 'd', 'r']
```

Comment:
This list is going to grow a lot and have to be maintained in parallel with the shortcuts themselves. I have two ideas of how to make it more maintainable:

1. Remove these and import the shortcuts directly in the tests.
2. Define these in a single object with keys that match the shortcut names.

(1) eliminates all redundancy, or (2) tidies it up.

Which one do you like better?

Reply:
I am not sure but I prefer the second option. I don't have strict reasons but I can say that I don't want our e2e test to be domain-dependent.

Think about this scenario, when somebody changes the gesture of the newThought by mistake, I want to catch this issue in our tests. 
I always think of the e2e tests from an end-user perspective. If you change the shortcut of any action, you need to inform me. Maybe you can mention this on the changelog or release description.  So, I think you need to inform our e2e test also.